test/WebJobs.Script.Tests.Integration/EventHubs/EventHubSamplesEndToEndTests.cs

```diff
@@ -26,7 +26,7 @@ public EventHubSamplesEndToEndTests(TestFixture fixture)
             _fixture = fixture;
         }
 
-        [Fact]
+        [Fact(Skip = "Pending extension installation updates")]
```

Comment:
@mathewc - I'm skipping these tests for now. Installation is working, but the process is not currently writing the types as we just make them all internal. Until we change the tooling to use the startup types, we need to keep this disabled.

Reply:
Can we add a TODO or a link to issue to make sure we dont miss enabling these tests?

Reply:
Will do. All skipped tests also get logged as skipped and we're going through all of them